hdl/XSimScript.tmpl

```diff
@@ -2,87 +2,78 @@
 
 set -eu -o pipefail
 
-{{ $name := .Name }}
+{{ if .Verbose }}
+set -x
+{{ end }}
+
 {{ $outDir := .OutDir }}
 {{ $outScript := .OutScript }}
```

Comment:
I'm still quite new at go. Why do you need to do this for those variables but not for the others? Can you point me to some resources to familiarize?

Reply:
This is how loops in the templates work. The dot is replaced with the current item and I need these two globals inside of the loop.